test/fuzz/mempool/checktx.go

```diff
@@ -17,7 +17,7 @@ func init() {
 	app := kvstore.NewApplication()
 	cc := abciclient.NewLocalCreator(app)
 	appConnMem, _ := cc(log.NewNopLogger())
-	err := appConnMem.Start()
+	err := appConnMem.Start(context.TODO())
```

Comment:
I'm fine with a TODO, but doesn't this mean this can't shut down at all (given that Stop is now gone)?

Reply:
it's in an init function in the fuzz tester.... so there's not really a great solution